src/cdk/a11y/key-manager/tree-key-manager.spec.ts

```diff
@@ -182,8 +182,18 @@ describe('TreeKeyManager', () => {
           itemList.notifyOnChanges();
         });
 
-        it('initializes with the first item activated', () => {
-          expect(keyManager.getActiveItemIndex()).withContext('active item index').toBe(0);
+        it('does not initialize with the first item activated', () => {
+          expect(keyManager.getActiveItemIndex()).withContext('active item index').toBe(-1);
```

Comment:
Wait, I thought we *do* want disabled options to be keyboard focusable? That's to align with WAI ARIA recommendation.

https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#focusabilityofdisabledcontrols

Reply:
hmm... true. do we still want to ignore clicks as well? or rather, should clicking a disabled item still focus it

Reply:
I see that https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#focusabilityofdisabledcontrols recommends that disabled treeitems be focusable, but I don't think it says anything about keyboard vs mouse focus. I'll see if I can look more to see if there's a recommendation for mouse focus.

Reply:
I think ultimately we can try it as-is (we ignore a click if it's disabled), and see if anyone reports a bug?